aioaws/core.py

```diff
@@ -21,6 +22,7 @@
 _AWS_AUTH_REQUEST = 'aws4_request'
 _CONTENT_TYPE = 'application/x-www-form-urlencoded'
 _AUTH_ALGORITHM = 'AWS4-HMAC-SHA256'
+_BUCKET_DOMAIN = '([^.]+).s3.[^.]+.amazonaws.com'
```

Comment:
This isn't quite right, the idea is that aioaws can generate upload links for buckets routed via a CNAME record to a custom domain like `mycdn.example.com`.

There shouldn't be much point in using teh full bucket domain like this case.